css/properties/mask-border-outset.json

```diff
@@ -0,0 +1,109 @@
+{
+  "css": {
+    "properties": {
+      "mask-border-outset": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/mask-border-outset",
+          "support": {
+            "chrome": [
+              {
+                "version_added": false
+              },
+              {
+                "alternative_name": "-webkit-mask-box-image-outset",
+                "version_added": "1"
+              }
+            ],
```

Comment:
When there's a support statement with `alternative_name`, then the `{ "version_added": false }` support statements isn't needed. Can you remove all such cases in this PR?

Reply:
The other files are fixed, but this one still has the unneeded `{ "version_added": false }` statements. We can merge as soon as that's done. Thank you!

Reply:
sorry about that - missed one - should be fixed now.